test/service/overseas.entities.service.spec.ts

```diff
@@ -105,7 +105,6 @@ describe(`Update Overseas Entity Service test suite`, () => {
 });
 
 describe(`Get Overseas Entity Service test suite`, () => {
-
```

Comment:
This change can be reverted as its removing a line

Reply:
Reverted to what is on the main branch